docs/sources/salesforce/example-api-responses/sanitized/composite_task_and_events.json

```diff
@@ -0,0 +1,141 @@
+{
+  "compositeResponse":[
+    {
+      "body":{
+        "totalSize":2,
+        "done":true,
+        "nextRecordsUrl":"/services/data/v57.0/query/01gRO0000016PIAYA2-500",
+        "records":[
+          {
+            "attributes":{
+              "type":"Task"
+            },
+            "Id":"00T5Y00007Txy7DUAR",
+            "WhoId":"null",
```

Comment:
:yellow_circle: in the original whoId is `null`, but here becomes a string `"null"`.
That doesn't seem right, should pass through `null`.
Something off in the psedonymization null handler (probably unrelated to this PR)